### PR TITLE
Add a link to the reproducible example repo in the bug report template

### PR DESCRIPTION
**Pull request by jeanbisutti**

### CODE REVIEW

**Review by trask — APPROVED** [review]
👍

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -11,10 +11,8 @@ Describe what you expected to happen.
 Describe what actually happened that was unexpected.
 
 ### To Reproduce
-Steps to reproduce the behavior:
 
-#### Sample Application
-If applicable, provide a sample application which reproduces the issue.
+An example reproducing the issue in [this repository](https://github.com/microsoft/ApplicationInsights-Java-Repros) would make it easier for the Application Insight maintainers to help you.
```
Comment (trask) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
Submitting a PR with an example reproducing the issue in [this repository](https://github.com/microsoft/ApplicationInsights-Java-Repros) would make it easier for the Application Insight maintainers to help you.
```

Comment (heyams) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: can we mention fork the repo and then submit a PR?

Comment (jeanbisutti) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @heyam I have pushed an update.

**Review by heyams — APPROVED** [review]
👍

**Pull request merged**